server/src/repositories/test_repository.ts

```diff
@@ -0,0 +1,30 @@
+import { inject, injectable } from "inversify";
+import { Pool } from "pg";
+import { TestResults } from "../entities/test";
+
+@injectable()
+export class TestRepository {
+    constructor(@inject("DBConnectionPool") private readonly pool: Pool) {}
+
+    async addTestResult(testResults: TestResults) {
```

Comment:
Keep with the standard repository pattern naming conventions

```suggestion
    async insertTestResult(testResult: TestResult) {
```